File: [6주차]/2750.py
n=int(input())
m=[]

for i in range(n):
    m.append(int(input()))
    
# insert sort
for i in range(1, len(m)):
    key=m[i]
    j=i-1
    while(j>=0) and (m[j]>key):
        m[j+1]=m[j]
        j-=1
    m[j+1]=key
        
for n in m:
    print (n)

File: week06/2750.js
// 2024.05.07 풀이
// 수 정렬하기
// https://www.acmicpc.net/problem/2750

const fs = require("fs");
const path = "./input.txt";

const readline = require("readline");
const rl = readline.createInterface({
	// input: process.stdin,
	input: fs.createReadStream(path),
	output: process.stdout,
});

let input = [];

rl.on("line", function (line) {
	input.push(line);
}).on("close", function () {
	solution(input);
	process.exit();
});

function solution(input) {
	input.shift();
	input.sort((a, b) => a - b).map((el) => console.log(el));
}
